scripts/00401-yuceehuchey.py

```diff
@@ -1,2 +1,2 @@
-print("Hello World, this is Nduanya Uchenna with HNGi7 ID HNG-00401 for stage 2 task yuceehuchey@gmail.com")
+print("Hello World, this is Nduanya Uchenna with HNGi7 ID HNG-00401 using python for stage 2 task yuceehuchey@gmail.com")
 flush=True
```

Comment:
Please delete this line of code or use it correctly. 
i.e. `print("Hello world!", flush=True)`
It should be inside the print statements brackets.